### DISCUSSION TITLE
Autogenerated TMP files

### DISCUSSION THREAD

**Discussion by trashhacker** [AvaloniaUI/Avalonia#17494] [Q&A]
whats up with that autogenerated TMP files 
![tmp pndg](https://github.com/user-attachments/assets/12960e14-da78-44b8-b416-d0f1f6e8de8c)

**Comment by maxkatz6** [comment]
I don't remember of any place where these TMP files are generated.

**Comment by trashhacker** [comment]
Oh OK than ist's maybe from codeium ... Sorry
